docs/pages/bssw/bssw_styling_common.md

```diff
@@ -83,6 +83,44 @@ Certain content types on [bssw.io](https://bssw.io) do not require formal refere
 
 The decision to *allow* or *require* references is one that should be agreed upon by the author and EB members prior to developing the content. When references are to be used, we require authors to use the less intrusive [reference links](https://www.markdownguide.org/basic-syntax#reference-style-links) ([full spec](https://github.github.com/gfm/#reference-link)) and to follow the guidelines described [here](https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/ReferencesInMarkdownHybridApproach.md) where the [`wikize_refs.py`](https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy) tool can be helpful.
 
+## Nonstandard handling of Markdown
+
+The [bssw.io] site uses as custom tool to translate the Markdown in each `*.md` file into HTML.
+This tool deals with Markdown differently in several ways compared to other common Markdown renderers (such as GitHub and GitHub Pages).
+It is important to know about these differences up front to avoid problems once the `*.md` files are previewed on [preview.bssw.io] and finally published to [bssw.io].
+Here, the major differences in how Markdown is handled that we know about.
+
+### Name of page created from first section name
+
+The name of the page on the [bssw.io] site is derived from the first section name at the top of the `<base>.md` file and not the name of the `<base>.md` file itself.
+For example, the file `ATPESC.md` with the first section/title of `# Preparing the Next Generation of Supercomputer Users` is given the derived page name `preparing-the-next-generation-of-supercomputer-users` on the site and the file name is ignored.
+This is similar to how standard Markdown renderers create anchors for regular sections within a document (see below).
+This can cause conflicts when two or more different `*.md` files have different names or different paths but have the same title.
+These would map into the same translated page name and causes undefined behavior.
+(NOTE: Simply having the name of the `*.md` file match the title using some convention does not guarantee the avoidance of a conflict.
+That is, files in different subdirectories with the same name and same title will not cause any problems with Git, GitHub, or GitHub Pages, but can result in a conflict with the bssw.io translator.
```

Comment:
For a normal Jekyll based site, we'd refer to this process as `site generation`, `page generation` or maybe `html rendering`. Does it make sense to use same terminology here instead of, for example, `translator`?

Reply:
Sure, I will use 'generator'.

Reply:
I fixed the last of those in commit 20cf5f89.